packages/test-support/addon-test-support/test-helpers.js

```diff
@@ -39,4 +83,52 @@ export function renderCard(type, id, format, options = {}) {
       return render(hbs`{{cardstack-content event-isolated content=card format=format params=params }}`);
     }
   });
-}
\ No newline at end of file
+}
+
+/**
+ * This sets up a `cardstack-card-test` component that can be use to render
+ * cards in the QUnit test fixture:
+ *
+ * ```js
+ * await render(hbs`{{cardstack-card-test "works-detail" 123 format="embedded"}}`);
+ * ```
+ *
+ * The positional parameters are the card name and the ID. The component also
+ * supports optional `format` and `params` parameters.
+ */
+export function setupCardTestComponent(hooks) {
+  hooks.beforeEach(function() {
+    let CardTestComponent = Component.extend({
+      tagName: '',
+
+      // inputs
+      type: null,
+      id: null,
+      format: 'isolated',
+      params: null,
+
+      // filled by `getSpaceForCardTask`
+      space: null,
+
+      // derived data
+      card: readOnly('space.primaryCard'),
+      _params: computed('space', 'params', function() {
+        Object.assign({}, this.get('space.params'), this.params)
+      }),
+
+      getSpaceForCardTask: task(function*() {
+        let space = yield getSpaceForCard(this.type, this.id);
+        this.set('space', space);
+      }).on('didInsertElement').cancelOn('willDestroyElement'),
+    });
+
+    CardTestComponent.reopenClass({
+      positionalParams: ['type', 'id'],
+    });
+
+    this.owner.register('component:cardstack-card-test', CardTestComponent);
+
+    this.owner.register('template:components/cardstack-card-test',
+      hbs`{{#if card}}{{cardstack-content event-isolated content=card format=format params=params}}{{/if}}`);
```

Comment:
I assume `event-isolated` is copied from somewhere and could either be removed or should be dynamic?

Reply:
It's copied from the existing `renderCard()` helper. I don't know what it does and would prefer not to touch it for now.

Reply:
I think you can just remove it - `cardstack-content` doesn't have any positional parameters anyway and I think this is simply ignored.

Reply:
then why is `renderCard()` using it? /cc @habdelra

Reply:
think @marcoow is right, it looks like a copy/paste mistaketo me.

Reply:
remove with the latest commit